restaurants/models.py

```diff
@@ -69,6 +69,7 @@ class Dish(models.Model):
     name = models.CharField(max_length=100)
     price = models.DecimalField(max_digits=10, decimal_places=2)
     dish_type = models.CharField(max_length=20, choices=DISH_TYPE_CHOICES)
+    image = models.ImageField(upload_to='dish_photos/', null=True, blank=True)
```

Comment:
Create a separate PR to add this model field and the related migration. Model changes should be done in a separate commit.